src/components/common/Info/Info.jsx

```diff
@@ -0,0 +1,30 @@
+/*
+ * Copyright © 2020 EPAM Systems, Inc. All Rights Reserved. All information contained herein is, and remains the
+ * property of EPAM Systems, Inc. and/or its suppliers and is protected by international intellectual
+ * property law. Dissemination of this information or reproduction of this material is strictly forbidden,
+ * unless prior written permission is obtained from EPAM Systems, Inc
+ */
+
+import React from 'react'
+import PropTypes from 'prop-types'
+import getGenres from '../../../utils/getGenres'
+import { Raiting } from '../Raiting'
+
+export const Info = ({ film, className }) => {
```

Comment:
Also here
is it really needed to pass className in this component
are there any cases where the Info will look different ?